nix/flakebox.nix

```diff
@@ -585,6 +586,12 @@ rec {
       bin = "gateway-cln-extension";
     };
 
+  recoverytool = flakeboxLib.pickBinary
```

Comment:
`fedimint-recoverytool` ?

Also, we should fix: https://github.com/NixOS/nixpkgs/blob/c8dec6df384ce6f24866adafbe11219ea5106be6/pkgs/by-name/fe/fedimint/package.nix#L57

Reply:
Oh, ugly. That's what it's called in `Cargo.toml`. I think if we change it to `recoverytool` there it should work

Reply:
ah sorry, misinterpreted. `fedimint-recoverytool` is probably better

Reply:
Even in this file (flakebox.nix) there is `fedimint-dbtool` few lines above. But yeah - we should just make recoverytool look like `fedimint-dbtool` in all these aspects, from Cargo.toml to nix files.

Reply:
done